### PR TITLE
Seed Contributor Workshop Dir for UUIC Hackillinois

### PR DESCRIPTION
**Pull request by MadhavJivrajani**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Seed Contributor Workshop Dir for UUIC Hackillinois
- Add attendee instruction
- Add README with agenda and intros of facilitators
- Seed OWNERS files

**Which issue(s) this PR fixes**:

Fixes #
None

**Special notes for your reviewer**:
None

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```
/assign @palnabarun @nikhita 
/cc @RinkiyaKeDad @rajaskakodkar

### CODE REVIEW

File: remote/hackillinois-osf-2021/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs: https://go.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+  - MadhavJivrajani
+  - rajaskakodkar
```
Comment (nikhita) [inline remote/hackillinois-osf-2021/OWNERS:5]: @rajaskakodkar will need to join k-sigs to be included in the OWNERS file.

**Comment by nikhita** [comment]
Approving, feel free to lgtm once OWNERS file is fixed.

/approve

File: remote/hackillinois-osf-2021/OWNERS (hunk 1 above)
Comment (MadhavJivrajani) [reply remote/hackillinois-osf-2021/OWNERS:5]: Ah, good catch
Thanks!

**Pull request merged**